<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>F1G 3D Model Viewer</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="GLWin" name="openGLWidget">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>400</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="modelTab">
       <attribute name="title">
        <string>Model</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btnLoadModel">
          <property name="text">
           <string>Load Model</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="xLabel">
          <property name="text">
           <string>X Angle Rotation:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="xAngleSlider">
          <property name="minimum">
           <number>-90</number>
          </property>
          <property name="maximum">
           <number>90</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="yLabel">
          <property name="text">
           <string>Y Angle Rotation:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="yAngleSlider">
          <property name="minimum">
           <number>-90</number>
          </property>
          <property name="maximum">
           <number>90</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="zLabel">
          <property name="text">
           <string>Z Angle Rotation:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="zAngleSlider">
          <property name="minimum">
           <number>-90</number>
          </property>
          <property name="maximum">
           <number>90</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Scale</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="scaleSlider">
          <property name="maximum">
           <number>20</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>X Position</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="xPosSlider">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Y Position</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="yPosSlider">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Z Position</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="zPosSlider">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="cameraTab">
       <attribute name="title">
        <string>Camera</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>X Position</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="cameraXSlider">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Y Position</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="cameraYSlider">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Z Position</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="cameraZSlider">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="lightsTab">
       <attribute name="title">
        <string>Lights</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="cbDirectionalLight">
          <property name="text">
           <string>Directional Light</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Intensity</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="lightIntensitySlider">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>X Position</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="lightXpositionSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Y Position</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="lightYpositionSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Z Position</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="lightZPositionSlider">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="LightsTabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>Ambient</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="lbAmbientRValue">
              <property name="text">
               <string>R:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="slAmbientR">
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="value">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbAmbientGValue">
              <property name="text">
               <string>G:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="slAmbientG">
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="value">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbAmbientBValue">
              <property name="text">
               <string>B:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="slAmbientB">
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="value">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>Diffuse</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="lbDiffuseRValue">
              <property name="text">
               <string>R:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="slDiffuseR">
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="value">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbDiffuseBValue">
              <property name="text">
               <string>B:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="slDiffuseB">
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="value">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbDiffuseGValue">
              <property name="text">
               <string>G:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="slDiffuseG">
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="value">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_6">
           <attribute name="title">
            <string>Specular</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="lbSpecularRValue">
              <property name="text">
               <string>R:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="slSpecularR">
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="singleStep">
               <number>1</number>
              </property>
              <property name="value">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbSpecularGValue">
              <property name="text">
               <string>G:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="slSpecularG">
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="value">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbSpecularBValue">
              <property name="text">
               <string>B:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="slSpecularB">
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="value">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLWin</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">glwin.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
